NEWS.md

```diff
@@ -1,2 +1,10 @@
 # 1.0.0
 - Initial release
+
+## 1.0.1
```

Comment:
Convention is for newest releases to go at the top of the file

Reply:
Yeah, my bad >_< We'll have to fix it the next time around, since this is already in the 1.0.1 release